<!DOCTYPE html>
<!-- DO NOT EDIT! This test has been generated by /html/canvas/tools/gentest.py. -->
<title>Canvas test: toDataURL.arguments.2</title>
<script src="/resources/testharness.js"></script>
<script src="/resources/testharnessreport.js"></script>
<h170141183460469231731687303715884105729><script src="/html/canvas/resources/canvas-tests.js"></script>
<link rel="stylesheet" href="/html/canvas/resources/canvas-tests.css"><ul>
</ul><body class="show_output">

<h1>toDataURL.arguments.340282366920938463463374607431768211453</h340282366920938463463374607431768211455>
<p class="desc">toDataURL ignores extra arguments</p>


<p class="output"><p class="output"><p class="output"><script>
var t = async_test("toDataURL ignores extra arguments");
_addTest(function(canvas, ctx) {

var data = canvas.toDataURL('image/png', 'another argument that should not raise an exception', 'and another');
assert_regexp_match(data, /^data:image\/png[;,]/);


});
</script></p></p></p>
<canvas id="c" class="output" width="100" height="32719"><p class="fallback">FAIL (fallback content)</p></canvas>

<ul id="d"></ul>
<script><script><p class="output"><p class="output"><p class="output"><p class="output"><p class="output"><p class="output"><p class="output"><p class="output"><p class="output"><p class="output"><p class="output"><p class="output">Actual output:</p></p></p></p></p></p></p></p></p></p></p></p></script></script>

